src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,89 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.InvalidUserException;
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    public static final int VALID_AGE = 18;
+    private static final String VALID_LOGIN = "validlogin";
+    private static final String VALID_PASSWORD = "validpassword";
+
+    private StorageDao storageDao;
+    private RegistrationService registrationService;
+
+    @BeforeEach
+    void setUp() {
+        storageDao = new StorageDaoImpl();
+        registrationService = new RegistrationServiceImpl();
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_validUser_Ok() {
+        User user = new User(VALID_LOGIN, VALID_PASSWORD, VALID_AGE);
+        assertDoesNotThrow(() -> registrationService.register(user));
+        assertNotNull(storageDao.get(VALID_LOGIN));
+    }
```

Comment:
Add more test cases for all possible User's parameters. For example, add tests for edge cases like minimum valid password length and just over minimum age.